packages/api/src/external/commonwell/cq-bridge/coverage-enhancer-factory.ts

```diff
@@ -17,5 +17,6 @@ export function makeCoverageEnhancer(): CoverageEnhancer | undefined {
   }
   const cookieManager = new CookieManagerOnSecrets(cookieArn, Config.getAWSRegion());
   const cwManagementApi = makeApi({ cookieManager, baseUrl: cwManagementUrl });
-  return new CoverageEnhancerApiLocal(cwManagementApi);
+  const orgIdExcludeList = new Set<string>();
```

Comment:
Only place not using the real orgIdExcludeList. it would have made me need to make the function async, and then that would have required propogating changes and been a pain in the ass. This code is getting deprecated soon anyway so I think this is ok.

Reply:
Sounds good. Can we just add a TODO/note there indicating why we're passing an empty set instead of calling it? Could also include a link to this thread.